scripts/upgrade-db-0.3-to-1.0.js

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env node
+"use strict";
+
+// **********************************************************************
+// ** This file contains a template script for updating databases from **
+// ** version 0.3 to version 1.0. You will likely need to edit and     **
+// ** customise it for your application service.                       **
+// **********************************************************************
+
+var Promise = require("bluebird");
+var Datastore = require("nedb");
+Promise.promisifyAll(Datastore.prototype);
+var nopt = require("nopt");
+var path = require("path");
+var fs = require("fs");
+
+const ROOM_DB = "1.0-db/rooms.db";
+
+var opts = nopt({
+    "help": Boolean,
+    rooms: path,
+}, {
+    "h": "--help"
+});
+
+if (!opts.help && !opts.rooms) {
+    console.log("--rooms is required.");
+    opts.help = true;
+}
+
+if (opts.help) {
+    console.log(
+`Database Upgrade script (v0.3 => v1.0)
+--------------------------------------
+
+ Usage:
+   --rooms   The path to rooms.db. Required.`
+);
+process.exit(0);
+}
+
+// *********************************************************
+// ** You will probably want to customise these functions **
+// *********************************************************
+
+function generateMatrixEntry(opts)
```

Comment:
Please doc what properties exist in `opts`.

If they are all the same then you can do it once in `:43`, but as of right now I got no clue.